launchable/utils/session.py

```diff
@@ -14,47 +12,33 @@ def _session_file_dir() -> Path:
     return Path(os.environ.get(SESSION_DIR_KEY) or DEFAULT_SESSION_DIR).expanduser()
```

Comment:
This might be a result of the evolution of thinking, but I notice that it's no longer tied to the current directory. Is that intentional?

This reads to me that it's using a known fixed location in user's home directory, which means if multiple builds are running with the same user ID (which is common in Jenkins) different builds will cross wire.